gridcapa-cse-valid-app/src/main/java/com/farao_community/farao/cse_valid/app/CseValidNetworkShifter.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.farao_community.farao.cse_valid.app;
+
+import com.farao_community.farao.cse_valid.app.configuration.EicCodesConfiguration;
+import com.farao_community.farao.cse_valid.app.exception.CseValidShiftFailureException;
+import com.farao_community.farao.dichotomy.api.NetworkShifter;
+import com.farao_community.farao.dichotomy.api.exceptions.GlskLimitationException;
+import com.farao_community.farao.dichotomy.api.exceptions.ShiftingException;
+import com.farao_community.farao.dichotomy.shift.LinearScaler;
+import com.farao_community.farao.dichotomy.shift.SplittingFactors;
+import com.powsybl.glsk.api.GlskDocument;
+import com.powsybl.iidm.network.Network;
+import org.springframework.stereotype.Component;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * @author Oualid Aloui {@literal <oualid.aloui at rte-france.com>}
+ */
+
+@Component
+public class CseValidNetworkShifter {
```

Comment:
I think we have to think about naming for this whole class. NetworkShifterProvider maybe, and something like fullImport(), exportCornerInitial() (or exportCornerWithItalyFrance()) and exportCornerDichotomy() (or exportCornerWithAllCountries()) for each public method. I feel like it will be more concise and explicit.